falcon/http_status.py

```diff
@@ -30,27 +30,25 @@ class HTTPStatus(Exception):
         headers (dict): Extra headers to add to the response.
         text (str): String representing response content. Falcon will encode
             this value as UTF-8 in the response.
-        body (str): Deprecated alias to :attr:`text`. Will be removed in a future
-            Falcon version. :attr:`text` take precedence if provided.
 
     Attributes:
         status (str): HTTP status line, e.g. '748 Confounded by Ponies'.
         headers (dict): Extra headers to add to the response.
         text (str): String representing response content. Falcon will encode
             this value as UTF-8 in the response.
-        body (str): Deprecated alias to :attr:`text`. Will be removed in a future
-            Falcon version.
 
     """
 
     __slots__ = ('status', 'headers', 'text')
 
-    def __init__(self, status, headers=None, text=None, body=None):
+    def __init__(self, status, headers=None, text=None):
         self.status = status
         self.headers = headers
-        self.text = text if text is not None else body
+        self.text = text
 
     @property  # type: ignore
-    @deprecated('Please use text instead.', is_property=True)
     def body(self):
-        return self.text
+        raise RemovedError(
```

Comment:
I opted to raise this vs. just flat out remove the attribute so that people still get some guidance on what happened and know how to update their code (vs. blowing up with an attribute error).

Reply:
Yes, that's much more helpful than just a plain `AttributeError` :+1:

Reply:
I guess this is another thing that will be removed in a future release, but it should be fine for now